charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```
@@ -430,18 +447,33 @@ data:
         regex: __meta_kubernetes_service_label_(.+)
       - source_labels: [ __meta_kubernetes_pod_name ]
         target_label: pod
+      - target_label: __address__
+        replacement: kube-apiserver:443
+      - source_labels: [__meta_kubernetes_pod_name]
+        regex: (.+)
+        target_label: __metrics_path__
+        replacement: /api/v1/namespaces/kube-system/pods/${1}:9153/proxy/metrics
```

Can you use the port number from the Service discovery information like this?
If so, can you adapt the other scrape configs accordingly?
```suggestion
      - source_labels: [__meta_kubernetes_pod_name, __meta_kubernetes_pod_container_port_number]
        regex: (.+);(.+)
        target_label: __metrics_path__
        replacement: /api/v1/namespaces/kube-system/pods/${1}:${2}/proxy/metrics
```